src/minecraft/FilePathManager.h

```diff
@@ -1,26 +1,33 @@
 #pragma once
 
+#include "legacy/Core.h"
 #include "std/string.h"
 
+namespace Core {
+
 class FilePathManager {
 
 public:
 
     char filler[0x20];
 
-    FilePathManager(mcpe::string, bool);
+    /// @symbol _ZN4Core15FilePathManagerC2ERKNS_4PathEb
+    FilePathManager(Legacy::Pre_1_13::Core::Path const&, bool);
 
-    mcpe::string getRootPath() const;
+    const mcpe::string &getRootPath() const;
 
-    mcpe::string getUserDataPath() const;
+    const mcpe::string &getUserDataPath() const;
 
-    mcpe::string getWorldsPath() const;
+    const mcpe::string &getWorldsPath() const;
```

Comment:
Broken on armhf unknown error, perhabs Symbol exists.
Now avoid calling getWorldsPath in server impl